src/components/CharSelector.js

```diff
@@ -0,0 +1,24 @@
+import '../css/CharSelector.css';
+import { useState, useContext } from 'react';
+import CharContext from '../contexts/CharContext';
+
+const CharSelector = () => {
+  const { characters } = useContext(CharContext);
+  const [charSelected, setCharSelected] = useState(characters[0]);
+
+  return(
```

Comment:
This is awesome. We should definitely get it implemented in the future!

Reply:
Oh yes, that file snuck it's way in there...